zebra/zebra_routemap.c

```diff
@@ -359,26 +359,46 @@ ALIAS (no_match_ip_address_prefix_list,
 
 DEFUN (set_src,
        set_src_cmd,
-       "set src A.B.C.D",
+       "set src (A.B.C.D|X:X::X:X)",
```

Comment:
It's a special parameter inside of quagga. Check https://github.com/Azure/sonic-quagga/blob/debian/0.99.24.1/lib/command.h#L264